app/assets/javascripts/application.js

```diff
@@ -29,24 +29,81 @@ function div_show() {
 document.getElementById('abc').style.display = "block";
 }
 
-google.load("visualization", "1", {packages:["corechart"]});
-      google.setOnLoadCallback(drawChart);
-      function drawChart() {
-
-        var data = google.visualization.arrayToDataTable([
-          ['Task', 'Hours per Day'],
-          ['Work',     11],
-          ['Eat',      2],
-          ['Commute',  2],
-          ['Watch TV', 2],
-          ['Sleep',    7]
-        ]);
-
-        var options = {
-          title: 'My Daily Activities'
-        };
-
-        var chart = new google.visualization.PieChart(document.getElementById('piechart'));
-
-        chart.draw(data, options);
-      }
+function facebook_login() {
```

Comment:
Identifier 'facebook_login' is not in camel case.<br>Unmatched '{'.<br>'facebook_login' is defined but never used.